src/adapters/ActivitiesJSONAdapter.test.js

```diff
@@ -0,0 +1,50 @@
+import activities from './../data/activities';
+import ActivitiesJSONAdapter from './ActivitiesJSONAdapter';
+
+describe('Activities Adapter Interface', () => {
+  let activitiesJSONAdapter, activityID;
+
+  beforeEach(() => {
+    [activityID] = Object.keys(activities);
+    activitiesJSONAdapter = new ActivitiesJSONAdapter(activities);
+  });
+
+  test('getActivity() returns an observable', () => {
```

Comment:
Since `getActivity` is the only method in the adapter, we may not need this right now, but it would be good to group all of the `getActivity()` tests in their own describe block similar to the way the SDK does:

```js
describe('getActivity() functionality', () => {
  test('getActivity() returns an observable', () => {
```

Reply:
yes, will do that after we come up with additional method.